src/types/Summary/Demand.d.ts

```diff
@@ -16,9 +16,10 @@ export type EditableDemandSummary = {
 export type Demand = DemandSummary &
   EditableDemandSummary & {
     Mail: string;
-    'Distance au réseau': string;
+    'Distance au réseau': number;
     'N° de dossier': string;
     'en ZDP': string;
+    'Date de la demande': string;
```

Comment:
We should not use `Date de la demande` but always `Date demandes`

Reply:
I think so too. Maybe we should merge these two columns in our data sources because is it some confusing. Also, i fix that.